packages/tests/e2e/robot/tests/local/local_packaging.robot

```diff
@@ -0,0 +1,122 @@
+*** Variables ***
+${RES_DIR}                  ${CURDIR}/resources
+${ARTIFACT_DIR}             ${RES_DIR}/artifacts/odahuflow
+${RESULT_DIR}               ${CURDIR}/local_pack_results
+
+${INPUT_FILE}               ${RES_DIR}/request.json
+${DEFAULT_RESULT_DIR}       ~/.odahuflow/local_packaging/training_output
+
+${LOCAL_CONFIG}             odahuflow/local_packaging
+
+*** Settings ***
+Documentation       local run of trainings and packagings
+...                 with specs on cluster and host, accent on packagings
+Resource            ../../resources/keywords.robot
+Resource            ../../resources/variables.robot
+Variables           ../../load_variables_from_profiles.py    ${CLUSTER_PROFILE}
+Library             odahuflow.robot.libraries.utils.Utils
+Library             Collections
+Suite Setup         Run Keywords
+...                 Set Environment Variable  ODAHUFLOW_CONFIG  ${LOCAL_CONFIG}  AND
+...                 Login to the api and edge  AND
+...                 StrictShell  odahuflowctl --verbose config set LOCAL_MODEL_OUTPUT_DIR ${DEFAULT_RESULT_DIR}
+Suite Teardown      Run Keywords
+...                 Remove Directory  ${RESULT_DIR}  recursive=True  AND
+...                 Remove Directory  ${DEFAULT_RESULT_DIR}  recursive=True  AND
+...                 Remove File  ${LOCAL_CONFIG}  AND
+...                 StrictShell  odahuflowctl logout
+Force Tags          cli  local  packaging
+Test Timeout        120 minutes
+
+*** Keywords ***
+Run Local Packaging
+    [Teardown]  Shell  docker rm -f ${container_id.stdout}
+    [Arguments]  ${options}
+        ${pack_result}  StrictShell  odahuflowctl --verbose local packaging ${options}
+
+        Create File  ${RESULT_DIR}/pack_result.txt  ${pack_result.stdout}
+        ${image_name}    StrictShell  tail -n 1 ${RESULT_DIR}/pack_result.txt | awk '{ print $4 }'
+        Remove File  ${RESULT_DIR}/pack_result.txt
+
+        StrictShell  docker images --all
+        ${container_id}  StrictShell  docker run -d --rm -p 5001:5000 ${image_name.stdout}
+
+        Sleep  5 sec
+        StrictShell  docker container list -as -f id=${container_id.stdout}
+
+        ${MODEL_HOST}    Get local model host
+        ${result_model}  StrictShell  odahuflowctl --verbose model invoke --url ${MODEL_HOST}:5001 --json-file ${RES_DIR}/request.json
+        Should be equal as Strings  ${result_model.stdout}  ${WINE_MODEL_RESULT}
```

Comment:
As far as I know, this may fail one day, because JSON does not guarantee a key order. I'd suggest to evaluate the JSONs and compare them as dicts.

Reply:
Moreover, spaces around colons `:` and commas `,` also may cause a failed comparison. I'm planning to use another json-library in model wrapper. It will not generate any spaces in a serialized string.

Reply:
OK, good. I'm fixing it.